morpheus/modules/schemas/examples/llm/content_extractor_schema.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2022-2024, NVIDIA CORPORATION.
```

Comment:
Is it possible to move the example schemas out of the Morpheus library and under the `examples` folder?